_posts/2020-09-14-notes-on-building-python-for-windows-using-rbm.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Notes on building Python for Windows using RBM"
+author: yanmaani
+tags: [News]
+---
+
+Namecoin would like for the Tor Browser to support .bit domains.
```

Comment:
AFAIK Tor Browser does not have a `the` in its name.

Reply:
Amended. Do the Tor Project have the definite article, or would it just be 'That's how Tor Project build Tor Browser for Windows'?